.travis.yml

```
@@ -11,7 +11,7 @@ before_install:
 - MAX_LINE_LENGTH=100 bash ci/check_line_lengths.sh src/**/*.md
 install:
 - source ~/.cargo/env || true
-- cargo install mdbook --version 0.4.1
+- cargo install mdbook --version 0.4.3
```

I don't know if it makes sense, but if you want to get the latest semver-compatible version, I *think* it should be:

```suggestion
- cargo install mdbook --version '^0.4.3'
```

---

My concern is that it may lose the integration with `mdbook-linkcheck` if we don't check it explicitly. But I think it should be quite an minor issue, I'd be happy to accept that change :)

---

Well, patch releases shouldn't break things, right? But it might be a good idea to have the latest patch release of `mdbook-linkcheck` too.